src/kansei/fallen/node/fallen_determinant.cpp

```diff
@@ -31,25 +31,37 @@ namespace kansei::fallen
 {
 
 FallenDeterminant::FallenDeterminant(const DeterminantType & type)
-: fallen_status(FallenStatus::STANDUP), determinant_type(type), fallen_back_raw_limit(491.0),
-  fallen_front_raw_limit(458.0), fallen_right_raw_limit(519.0), fallen_left_raw_limit(498.0)
+: fallen_status(FallenStatus::STANDUP), determinant_type(type)
 {
 }
 
 void FallenDeterminant::load_config(const std::string & path)
 {
-  std::string file_name =
-    path + "imu/" + "kansei.json";
+  std::string file_name = path + "kansei.json";
   std::ifstream file(file_name);
+
+  if (!file.is_open()) {
+    throw std::runtime_error("Failed to open file: " + file_name);
+  }
+
   nlohmann::json imu_data = nlohmann::json::parse(file);
 
   for (const auto &[key, val] : imu_data.items()) {
-    if (key == "fallen_limit") {
+    if (key == "accel_limit") {
+      try {
+        val.at("accel_back_limit").get_to(accel_back_limit);
+        val.at("accel_front_limit").get_to(accel_front_limit);
+        val.at("accel_right_limit").get_to(accel_right_limit);
+        val.at("accel_left_limit").get_to(accel_left_limit);
+      } catch (nlohmann::json::parse_error & ex) {
+        std::cerr << "parse error at byte " << ex.byte << std::endl;
+      }
```

Comment:
You may need to ask older ICHIRO members why we happen to use try-catch when loading values from a JSON object like this. I suggest not just catching an exception and logging the error, it's better to let it crash than to continue the program with undefined behavior:

```suggestion
      try {
        val.at("accel_back_limit").get_to(accel_back_limit);
        val.at("accel_front_limit").get_to(accel_front_limit);
        val.at("accel_right_limit").get_to(accel_right_limit);
        val.at("accel_left_limit").get_to(accel_left_limit);
      } catch (nlohmann::json::parse_error & ex) {
        std::cerr << "Parse error at byte " << ex.byte << std::endl;
        throw ex;
      }
```

Or even better, maybe we don't need try-catch at all:

```suggestion
      val.at("accel_back_limit").get_to(accel_back_limit);
      val.at("accel_front_limit").get_to(accel_front_limit);
      val.at("accel_right_limit").get_to(accel_right_limit);
      val.at("accel_left_limit").get_to(accel_left_limit);
```